onnx2kerastl/reshape_layers.py

```diff
@@ -207,12 +207,15 @@ def target_layer(x):
                 input_0 = ensure_tf_type(layers[node.input[0]], name="%s_const" % keras_name)
                 input_0_shape = input_0.shape
                 first_mismatch = np.argmin(np.array(input_0_shape[:len(input_1)]) == input_1)
-                if (input_1 == None).any() and (np.array(input_0_shape) == None).any() and len(input_1) < len(input_0_shape)\
-                        and input_1[first_mismatch] == -1: #reshape end
+                if (input_1 == None).any() and (np.array(input_0_shape) == None).any() and len(input_1) < len(
+                        input_0_shape) \
+                        and input_1[first_mismatch] == -1:  # reshape end
                     end_match_arr = np.array(input_0_shape[-len(input_1):]) == input_1
                     end_idx_match = np.argmax((np.array(input_0_shape[-len(input_1):]) == input_1))
                     end_idx_match = end_idx_match + len(input_0_shape) - len(input_1) if end_idx_match > first_mismatch \
-                                                     and end_match_arr[end_idx_match] else len(input_0_shape) + 1
+                                                                                         and end_match_arr[
+                                                                                             end_idx_match] else len(
+                        input_0_shape) + 1
```

Comment:
Can you break it into a regular if-else clause? its a bit hard to follow when its so long